types/throng/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for throng 4.0
+// Project: https://github.com/hunterloftis/throng
+// Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function throng(options: throng.WorkerCallback | throng.Options | number, startFunction?: throng.WorkerCallback): void;
+declare namespace throng {
+    type WorkerCallback = (id: number) => void;
+
+    class Options {
```

Comment:
If `new throng.Options()` doesn't work at runtime, this should be an `interface`.